e2e-tests/e2e-fixtures/gef/mocks/mock-deals.js

```diff
@@ -0,0 +1,178 @@
+import Chance from 'chance';
+import { DEAL_STATUS, DEAL_SUBMISSION_TYPE, DEAL_TYPE } from '../constants';
+import { BANK1_MAKER1 } from '../../portal-users.fixture';
+
+const chance = new Chance();
+
+const eligibilityCriteria = (type) => ({
```

Comment:
This seems like it's repeating an existing file -- Is it possible to commonise both of these?

A bit of background: one of the existing bits of tech debt on this codebase is a lot of `mock bla` hanging around -- most of which aren't typed, and therefore require maintenance and can cause our e2es to inadvertently go out of date.

Reply:
It might be better to add a new ticket for that ?